tbot_contrib/gpio/gpio.py

```diff
@@ -0,0 +1,60 @@
+from tbot.machine import linux
+
+
+class gpio:
+    def __init__(self, lnx: linux.LinuxShell, gpio_number: str):
+        self._lnx = lnx
+        self._gpio_number = gpio_number
+        self._gpio_sysclass_path = "/sys/class/gpio/"
+        self._gpio = self._gpio_sysclass_path + "gpio" + self._gpio_number
+        self._export()
+
+    def _export(self) -> None:
+        output = self._lnx.exec0(
+            "find", self._gpio_sysclass_path, "-name", "gpio" + self._gpio_number
+        )
+        if output.strip() == (self._gpio):
+            pass
+        else:
+            self._lnx.exec0(
+                "echo",
+                self._gpio_number,
+                linux.Raw(">"),
+                self._gpio_sysclass_path + "export",
```

Comment:
Never use `linux.Raw()`! ;)  Instead, tbot has a helper for this usecase:

```python
self.host.exec0(
    "echo",
    str(self.gpio_number),
    linux.RedirStdout(self.host.fsroot / "sys/class/gpio/export"),
)
```

BUT, on latest `master` at least, there is an even nicer way to do this:

```python
(self.host.fsroot / "sys/class/gpio/export").write_text(str(gpio_number))
```